plugins/modules/vmware_object_role_permission.py

```diff
@@ -55,7 +55,8 @@
     default: 'Folder'
     choices: ['Folder', 'VirtualMachine', 'Datacenter', 'ResourcePool',
               'Datastore', 'Network', 'HostSystem', 'ComputeResource',
-              'ClusterComputeResource', 'DistributedVirtualSwitch']
+              'ClusterComputeResource', 'DistributedVirtualSwitch',
+              'DistributedVirtualPortgroup']
```

Comment:
Thanks, @snagoor for the patch!  
Rest LGTM

Could you please add a changelog file to `changelogs/fragments`?  
https://docs.ansible.com/ansible/latest/dev_guide/testing/sanity/changelog.html

If the fragments directory doesn't exist, please make it.

e.g. https://github.com/sky-joker/community.vmware/blob/55f0baa7f0c894fb3383c6dcfa19561b28cd4809/changelogs/fragments/915-vmware_host_datastore.yml

The above number of the file name is PR number.